### PR TITLE
Implemented logic for single sign-on(#1xx0wm1)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/views/login.vue [hunk 1]
```diff
@@ -59,11 +59,23 @@ export default defineComponent({
...
-    })
+      currentInstanceUrl: 'user/getInstanceUrl',
+      token: 'user/getUserToken'
+    }),
   },
-  mounted() {
-    this.instanceUrl= this.currentInstanceUrl;
+  async mounted() {
+    this.instanceUrl = this.currentInstanceUrl;
+    console.log(this.$route);
```
Comment (ymaheshwari1) [inline src/views/login.vue:68]: Remove console statement

File: src/views/login.vue [hunk 2]
```diff
@@ -59,11 +59,23 @@ export default defineComponent({
...
+    if(this.$route.redirectedFrom?.query.info){
+      console.log(this.$route.redirectedFrom?.query.info)
```
Comment (ymaheshwari1) [inline src/views/login.vue:70]: Remove console statement.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]
Add changelog entry.

**Comment by disha1202** [comment]
> Add changelog entry.

Added changelog entry.

File: src/views/login.vue (hunk 2 above)
Comment (disha1202) [reply src/views/login.vue]: Removed.

File: src/views/login.vue (hunk 1 above)
Comment (disha1202) [reply src/views/login.vue]: Removed.

**Comment by k2maan** [comment]
Closing. Will be handled later when we pickup the commerce hub.

**Pull request closed without merging**